homeassistant/components/sensor/launch.py

```diff
@@ -0,0 +1,92 @@
+"""
+A sensor platform that give you information about the next space launch.
+
+For more details about this platform, please refer to the documentation at
+https://www.home-assistant.io/components/sensor.launch/
+"""
+from datetime import timedelta
+import logging
+
+import voluptuous as vol
+
+import homeassistant.helpers.config_validation as cv
+from homeassistant.components.sensor import PLATFORM_SCHEMA
+from homeassistant.const import ATTR_ATTRIBUTION, CONF_NAME
+from homeassistant.helpers.entity import Entity
+from homeassistant.helpers.aiohttp_client import async_get_clientsession
+
+REQUIREMENTS = ['pylaunches==0.1.1']
+
+_LOGGER = logging.getLogger(__name__)
+
+CONF_ATTRIBUTION = "Data provided by Launch Library."
```

Comment:
Sorry, I just noticed one more thing: the `CONF_` prefix is for configuration keys so I dont think it should be used here.